src/styles/global.scss

```diff
@@ -9,4 +9,9 @@ body {
     color: $dark-green;
     font-weight: 48px;
   }
+
+  .mt-5.container {
+    margin-top: 0 !important;
+    padding-top: 4rem;
+  }
```

Comment:
Does this change the spacing at the top of every section?

Reply:
yes but basically what I did was replace the margin style that was there w the equivalent padding instead so that it fixes the nav scroll thing on all the sections